src/sentry/tasks/files.py

```diff
@@ -86,3 +87,34 @@ def delete_unreferenced_blobs(blob_model, blob_index_model, blob_ids):
                 # Do nothing if the blob was deleted in another task, or
                 # if had another reference added concurrently.
                 pass
+
+
+@instrumented_task(
+    name="sentry.tasks.files.copy_to_control",
+    queue="files.copy",
```

Comment:
You'll need to add a Queue() definition in `server.py` if you want to have a dedicated queue for this.

Reply:
I wasn't sure about this. We definitely need _a_ queue. How do workers actually consume from the queue? Can we manage concurrency? Should I just use `files.delete`?

Reply:
Some notes around adding new queues and workers are here: https://www.notion.so/sentry/RabbitMQ-0b28fcb3a2174969b3c134d66550cb6c?pvs=4#e9973d5eb4aa4745ae78f5b88f9f4c9b

Just a note that the federation part is outdated and can be ignored, I'll just double check first before removing it.

Reply:
> How do workers actually consume from the queue? Can we manage concurrency? Should I just use files.delete?

If you don't define an explicit queue, celery will put messages in the 'default' queues which are consumed by the worker.glob consumer groups. Celery uses 'competing consumers' with the goal of burning queue backlogs as quickly as possible, there are no limitations on concurrency other than the number of workers and number of tasks.

Having a dedicated queue would allow us to be more precise in the number of workers dedicated to this task backlog. We could also gate the production of messages with a flagr feature/option based feature that we gradually increase so that we don't get a thundering herd when this initially deploys.